src/Components/Server/src/Builder/ServerComponentsEndpointOptions.cs

```diff
@@ -22,23 +22,35 @@ public class ServerComponentsEndpointOptions
     /// <para>
     /// A content security policy provides defense against security threats that can occur if
     /// the app uses compression and can be embedded in other origins. When compression is enabled,
-    /// embedding the app inside an <c>iframe</c> from other origins is prohibited.
+    /// embedding the app inside an <c>iframe</c> from other origins is forbidden.
     /// </para>
     /// <para>
     /// For more details see the security recommendations for Interactive Server Components in
     /// the official documentation.
     /// </para>
     /// </remarks>
-    public string? ContentSecurityFrameAncestorPolicy { get; set; } = "'self'";
+    public string? ContentSecurityFrameAncestorsPolicy { get; set; } = "'self'";
+
+    /// <summary>
+    /// Gets or sets a value that determines if the websocket compression should be disabled.
+    /// </summary>
+    /// <remarks>
+    /// Websocket compression is enabled by default, but it can be disabled by setting this value to <see langword="true" />.
+    /// When a callback for <see cref="ConfigureWebSocketAcceptContext"/> is provided, the value of this property will be
+    /// ignored, whether compression is enabled or not will be determined by the callback, and the Content Security Policy
+    /// will be applied according to the value of <see cref="ContentSecurityFrameAncestorsPolicy"/>.
```

Comment:
I may be misremembering, but I thought the conclusion during API review was that the value for `DisableWebSocketCompression` would specify the value of the `acceptContext`'s `DangerousEnableCompression` property before the `acceptContext` gets passed to `ConfigureWebSocketAcceptContext`. So, this property doesn't get ignored if a callback gets specified, but the callback allows additional configuration on top of the compression setting. Does that sound right?

Reply:
It actually looks like this PR implements that behavior already - maybe the comment just needs updating?

Reply:
It was the other way around (It's not captured in the notes, but in the actual discussion in the feed) the callback always wins.